controller/Publish.php

```diff
@@ -58,12 +58,21 @@ public function publish(){
             $testUri = $this->getRequestParameter('testUri');
             $classUri = $this->getRequestParameter('classUri');
             $test = $this->getResource($testUri);
+
+            $testService = \taoTests_models_classes_TestsService::singleton();
```

Comment:
Small notice: when I use old `OntologyClassService`s I prefer to retrieve it like:
```
$this->getServiceLocator()->get(\taoTests_models_classes_TestsService::class)
```
(knowing that `singleton()` call does the same).
It looks more uniform with other services retrieval and allows you to mock them in tests.